c2corg_ui/templates/helpers/view.html

```diff
@@ -200,6 +177,39 @@ <h4 translate>No maps associated</h4>
   % endif
 </%def>
 
+<%def name="test_association_rights(document)">\
+  % if is_collaborative(document):
+    userCtrl.auth.isAuthenticated()
+    && userCtrl.auth.isModerator()
+  % elif is_personal(document):
+    userCtrl.hasEditRights(
+        '${get_doc_type(document['type'])}',
+        {'users': ${dumps(document['associations']['users'])},
+      % if document['type'] == 'c':
+        'articleType': '${document['article_type']}',
+        'authorId': '${document['author']['user_id']}'
+      % elif document['type'] == 'i':
+        'imageType': '${document['image_type']}',
+        'imageCreator': '${document['creator']['user_id']}',
+      % endif
+    })
+  % endif
+</%def>
+
+<%def name="check_if_any_associations(doc)">\
```

Comment:
Argh
Then you get in the HTML something like
```
ng-if="true || userCtrl.hasEditRights( 'articles', { 'articleType': 'personal', 'authorId': 169325 })"
```
Berk! :P 

Please rather create a real python function here https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/utils/__init__.py

and call it in the mako helper to figure if the block should added or not.